controlplane/src/bin/seed-billing.ts

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+import 'dotenv/config';
+
+import { readFileSync } from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { drizzle } from 'drizzle-orm/postgres-js';
+import postgres from 'postgres';
+import * as schema from '../db/schema.js';
+
+const databaseConnectionUrl = process.env.DB_URL || 'postgresql://postgres:changeme@localhost:5432/controlplane';
+
+const seedBilling = async () => {
+  const queryConnection = postgres(databaseConnectionUrl);
+  const db = drizzle(queryConnection, { schema: { ...schema } });
+
+  const configPath = process.argv[2]
+    ? path.resolve(process.cwd(), process.argv[2])
+    : path.join(path.dirname(fileURLToPath(import.meta.url)), 'billing.json');
+
+  const data = readFileSync(configPath, 'utf8');
+  const json = JSON.parse(data);
+
+  const entries = Object.entries<{
```

Comment:
We should validate the JSON file with a validator e.g ZOD.

Reply:
Good idea